pkg/cmd/roachtest/tests/predecessor_version.go

```diff
@@ -36,7 +36,8 @@ func PredecessorVersion(buildVersion version.Version) (string, error) {
 	// checkpoint option enabled to create the missing store directory
 	// fixture (see runVersionUpgrade).
 	verMap := map[string]string{
-		"21.2": "21.1.8",
+		"22.1": "21.2.0-beta.2",
```

Comment:
I think you need to rename the fixture files to `checkpoint-v21.2.0-beta.2`, the snippet that moved them there isn't advanced enough to figure out the current situation